vyxal/helpers.py

```diff
@@ -43,3 +298,72 @@ def uncompress_str(string: str) -> str:
 def uncompress_num(num: str) -> int:
     # TODO (lyxal) Implement number (un)compression
     raise NotImplementedError()
+
+
+def vy_eval(item: str, ctx: context.Context) -> Any:
+    """Evaluates an item. Does so safely if using the online
+    interpreter"""
+
+    if ctx.online:
+        try:
+            return ast.literal_eval(item)
+        except:
+            # TODO: eval as vyxal
+            return item
+    else:
+        try:
+            return eval(item)
+        except:
+            return item
+
+
+def vy_str(item: Any, ctx: context.Context) -> str:
+    """Convert to string, using custom vyxal formatting"""
+    if type(item) is LazyList:
+        item = list(item)
+
+    if type(item) is list:
+        return "⟨" + "|".join([vy_str(y) for y in x]) + "⟩"
+
+    return str(item)
+
+
+def vy_zip(*items) -> list:
+    """Like python's zip, but fills shorter lists with 0s"""
+
+    items = list(map(iter, items))
+    while True:
+        ret = []
+        exhausted_count = 0
+        for item in items:
+            try:
+                ret.append(next(item))
+            except:
+                ret.append(0)
+                exhausted_count += 1
+
+        if len(items) == exhausted_count:
+            break
+
+        yield ret
+
+
+def wrap(vector: Union[str, list], width: int) -> List[Any]:
+    # Because textwrap.wrap doesn't consistently play nice with spaces
```

Comment:
Might want to make this a nicer docstring instead?

Reply:
That comment is deadstuff from v2.4